### PR TITLE
Gillian Chan section 11

### PR DESCRIPTION
**Pull request by GillChanSG**
Please fill out the survey before submitting the pull request. Thanks!

🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀

How many hours did you spend on this assignment?
ages! few days.

Please fill in one error and/or error message you received while working on this assignment.
unexpected end of input 

What part of the assignment did you spend the most time on?
starting it

Comfort Level (1-5):
1

Completeness Level (1-5):
1.5

What did you think of this deliverable?
I have learnt a lot from Michelle on how to think of it logically.

Is there anything in this code that you feel pleased about?
Getting it to work

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1,4 +1,36 @@
-var main = function (input) {
-  var myOutputValue = 'hello world';
+var getRandomSelection = function () {
+  var randomNum = Math.floor(Math.random() * 3);
+  if (randomNum == 0) {
+    return scissors;
```
[the remark this answers is not in the document]
Comment (michelle-mok) [reply script.js:4]: scissors need to be in quotation marks e.g., 'scissors'

File: script.js [hunk 2]
```diff
@@ -1,4 +1,36 @@
...
+  }
+  if (randomNum == 1) {
+    return paper;
```
[the remark this answers is not in the document]
Comment (michelle-mok) [reply script.js:7]: paper needs to be in quotation marks

File: script.js [hunk 3]
```diff
@@ -1,4 +1,36 @@
...
+  }
+  return stone;
```
[the remark this answers is not in the document]
Comment (michelle-mok) [reply script.js:9]: stone needs to be in quotation marks

**Review by michelle-mok — COMMENTED** [review]
code is compact without repetition, well done !

File: script.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (GillChanSG) [reply script.js:4]: Noted. Thanks @michelle-mok

File: script.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (GillChanSG) [reply script.js:7]: Got it thanks.

File: script.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (GillChanSG) [reply script.js:9]: Got it thanks.

File: script.js [hunk 4]
```diff
@@ -1,4 +1,36 @@
...
+    playerInput != "paper" &&
+    playerInput != "stone"
+  ) {
+    myOutputValue = `Please type either 'scissors', 'paper' or 'stone' to start the game.`;
+  } else if (playerInput == computerInput) {
+    myOutputValue = `You have input ${playerInput} and computer has input ${computerInput}. It's a draw.`;
+  } else if (
+    (playerInput == "scissors" && computerInput == "paper") ||
+    (playerInput == "paper" && computerInput == "stone") ||
+    (playerInput == "stone" && computerInput == "scissors")
```
[the remark this answers is not in the document]
Comment (GillChanSG) [reply script.js:29]: Hi @michelle-mok , what does this mean? I don't understand where to put in the single quote.

**Pull request closed without merging**